src/main/java/de/uhd/ifi/se/decision/management/jira/model/KnowledgeStatus.java

```diff
@@ -21,18 +24,46 @@ public static KnowledgeStatus getKnowledgeStatus(String status){
 
 	public static List<String> toList() {
 		List<String> knowledgeStatus = new ArrayList<>();
-		for(KnowledgeStatus status: KnowledgeStatus.values()){
+		for (KnowledgeStatus status : KnowledgeStatus.values()) {
 			knowledgeStatus.add(status.toString());
 		}
 		return knowledgeStatus;
 	}
 
 	@Override
-	public String toString(){
-		return this.name().substring(0,1).toUpperCase(Locale.ENGLISH)
+	public String toString() {
+		return this.name().substring(0, 1).toUpperCase(Locale.ENGLISH)
 				+ this.name().substring(1).toLowerCase(Locale.ENGLISH);
 	}
 
+	public static KnowledgeType getNewKnowledgeTypeForStatus(KnowledgeStatus status, KnowledgeType formerType) {
+		if (status == null || formerType == null) {
+			return KnowledgeType.OTHER;
+		}
+		if (formerType == KnowledgeType.DECISION) {
+			if (status == IDEA || status == DISCARDED) {
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [These nested if statements could be combined](https://app.codacy.com/app/UHD/cures-condec-jira/pullRequest?prid=4636195)